sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/DerivedMetric.cs

```diff
@@ -22,15 +22,15 @@ internal class DerivedMetric<TTelemetry>
 
         private static readonly MethodInfo DoubleParseMethodInfo = typeof(double).GetMethod(
            "Parse",
-           new[] { typeof(string), typeof(IFormatProvider) });
+           new[] { typeof(string), typeof(IFormatProvider) })!;
```

Comment:
Documentation says `GetMethod()` may return null.
But these are well known methods on primitive types. 
I think it's safe to specify `!` here.